_projects/2023/100407986/100407986.Rmd

```diff
@@ -0,0 +1,459 @@
+---
+title: "Change in abortion after Dobbs"
+description: |
+  A visualization of how abortions changed after Dobbs
```

Comment:
This is in the title. It would be nice to have some more context about what's Dobbs, etc. here. ;-)